recipes/thriftpy/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+
+$PYTHON setup.py build_ext
```

Comment:
Again not a big deal. So, if it works for you, I am fine with it. Thanks for explaining. :+1: